packages/features/ee/workflows/lib/reminders/templates/emailReminderTemplate.ts

```diff
@@ -1,43 +1,50 @@
+import { WorkflowActions } from "@prisma/client";
+
 import dayjs from "@calcom/dayjs";
 import { APP_NAME } from "@calcom/lib/constants";
 
 const emailReminderTemplate = (
-  startTime: string,
-  endTime: string,
-  eventName: string,
-  timeZone: string,
-  attendee: string,
-  name: string
+  isEditingMode: boolean,
+  action?: WorkflowActions,
+  startTime?: string,
+  endTime?: string,
+  eventName?: string,
+  timeZone?: string,
+  attendee?: string,
+  name?: string,
+  isBrandingDisabled?: boolean
 ) => {
-  const emailSubject = `Reminder: ${eventName} - ${dayjs(startTime)
-    .tz(timeZone)
-    .format("ddd, MMM D, YYYY")} ${dayjs(startTime).tz(timeZone).format("H:mmA")}`;
-
-  const templateBodyText = `Hi ${name},this is a reminder that your meeting (${eventName}) with ${attendee} is on ${dayjs(
-    startTime
-  )
-    .tz(timeZone)
-    .format("YYYY MMM D")} at ${dayjs(startTime).tz(timeZone).format("h:mmA")} ${timeZone}.`;
+  let eventDate = "";
+
+  if (isEditingMode) {
+    endTime = "{EVENT_END_TIME}";
+    eventName = "{EVENT_NAME}";
+    timeZone = "{TIMEZONE}";
+    attendee = action === WorkflowActions.EMAIL_ATTENDEE ? "{ORGANIZER}" : "{ATTENDEE}";
+    name = action === WorkflowActions.EMAIL_ATTENDEE ? "{ATTENDEE}" : "{ORGANIZER}";
+    eventDate = "{EVENT_DATE_ddd, MMM D, YYYY H:mmA}";
```

Comment:
nit: I think we should abstract these variables out so that we can maintain consisteny b/w email and sms reminders.